lib/pg_audit_log/triggers.rb

```diff
@@ -42,6 +42,13 @@ def install
         end
       end
 
+      def uninstall_extra
```

Comment:
what do you think about using "stale" instead of "extra" here? i find "stale" a little better indicates uninstalling triggers that had been added previously, but are no longer needed

Reply:
Oh yeah, `stale` is way better, will change